CONTRIBUTING.md

```diff
@@ -20,6 +18,33 @@ languages to enrich the options available to programmers.
 - [https://projects.eclipse.org/projects/technology.omr](https://projects.eclipse.org/projects/technology.omr)
 
 
+## GitHub Issues
+
+This project uses GitHub Issues to track ongoing development and issues. Be sure
+to search for existing bugs before you create another one. Contributions are always welcome!
+
+- [https://github.com/eclipse/omr/issues](https://github.com/eclipse/omr/issues)
+
+## Submitting a contribution
+You can propose contributions by sending pull requests through GitHub. Following these guidelines
+will help us to merge your pull requests smoothly:
```

Comment:
The markdown formatting isn't quite right - it's messed up when I preview the file.  I think you might need to add blank lines between paragraphs, and maybe outdent the commit dtemplate.